phpunit/class-wp-theme-json-resolver-test.php

```diff
@@ -4,6 +4,8 @@
  * Test WP_Theme_JSON_Resolver_Gutenberg class.
  *
  * @package Gutenberg
+ *
+ * @since 5.8.0
  */
 
 class WP_Theme_JSON_Resolver_Gutenberg_Test extends WP_UnitTestCase {
```

Comment:
In core this is called `Tests_Theme_wpThemeJsonResolver`. Should we rename this to `Tests_Theme_wpThemeJsonResolver_Gutenberg`?

Reply:
I wonder if we should match the file path too -> `tests/phpunit/tests/theme/wpThemeJsonResolver.php`